ttnn/cpp/ttnn/reports.hpp

```diff
@@ -64,10 +64,6 @@ struct BufferInfo {
 std::vector<BufferInfo> get_buffers() {
     std::vector<BufferInfo> buffer_infos;
     for (const auto &[key, buffer] : tt::tt_metal::detail::BUFFER_MAP.value()) {
-        if (not buffer->is_l1()) {
-            continue;
-        }
```

Comment:
why?

Reply:
This is unrelated to this PR. I am just trying to sneak it in because this is needed for the visualizer